etc/spack/defaults/config.yaml

```
@@ -190,3 +190,8 @@ config:
   # Set to 'false' to allow installation on filesystems that doesn't allow setgid bit
   # manipulation by unprivileged user (e.g. AFS)
   allow_sgid: true
+
+  # Whether to set the terminal title to display status information during
+  # building and installing packages. This gives information about Spack's
+  # current progress as well as the current and total number of packages.
+  set_terminal_title: false
```

```suggestion
  terminal_title: false
```

which is more in line with `locks: true`, `ccache: false` etc.

---

Well, we also have options with verbs like `verify_ssl` and `allow_sgid`, but I don't have any preference.